gov.pnnl.goss.gridappsd/applications/fncsgossbridge/apps/fncs_goss_bridge.py

```diff
@@ -1,55 +1,26 @@
-# the following disclaimer in the documentation and/or other materials provided with the distribution.
-# Other than as used herein, neither the name Battelle Memorial Institute or Battelle may be used in any 
-# form whatsoever without the express written consent of Battelle. 
-# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS “AS IS” AND ANY 
-# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF 
-# MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL 
-# BATTELLE OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, 
-# OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE 
-# GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED 
-# AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING 
-# NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED 
-# OF THE POSSIBILITY OF SUCH DAMAGE.
-# General disclaimer for use with OSS licenses
-# 
-# This material was prepared as an account of work sponsored by an agency of the United States Government. 
-# Neither the United States Government nor the United States Department of Energy, nor Battelle, nor any 
-# of their employees, nor any jurisdiction or organization that has cooperated in the development of these 
-# materials, makes any warranty, express or implied, or assumes any legal liability or responsibility for 
-# the accuracy, completeness, or usefulness or any information, apparatus, product, software, or process 
-# disclosed, or represents that its use would not infringe privately owned rights.
-# 
-# Reference herein to any specific commercial product, process, or service by trade name, trademark, manufacturer, 
-# or otherwise does not necessarily constitute or imply its endorsement, recommendation, or favoring by the United 
-# States Government or any agency thereof, or Battelle Memorial Institute. The views and opinions of authors expressed 
-# herein do not necessarily state or reflect those of the United States Government or any agency thereof.
-# 
-# PACIFIC NORTHWEST NATIONAL LABORATORY operated by BATTELLE for the 
-# UNITED STATES DEPARTMENT OF ENERGY under Contract DE-AC05-76RL01830
-#-------------------------------------------------------------------------------
 '''
 Created on Jan 6, 2017
 
 @author: fish334
 @author: poorva1209
 '''
-import fncs
+import os
 import json
-import yaml
 import sys
 import time
+from datetime import datetime
+
 import stomp
-import logging 
+import yaml
+
+try:
+    import fncs
+except:
+    if not os.envrion.get("CI"):
+        raise ValueError("fncs.py is unavailable on the python path.")
+    else:
+        sys.stdout.write("Running tests.\n")
+
 
 input_from_goss_topic = '/topic/goss/gridappsd/fncs/input' #this should match GridAppsDConstants.topic_FNCS_input
```

Comment:
Consistency...we need to use either snake case or camel case within the project...not both.